package.json

```diff
@@ -86,10 +86,10 @@
     "@babel/preset-env": "^7.16.11",
     "@babel/runtime": "^7.17.8",
     "@influxdata/oats": "0.5.3",
-    "@testing-library/dom": "^8.11.1",
-    "@testing-library/jest-dom": "^5.15.0",
-    "@testing-library/react": "^12.1.2",
-    "@testing-library/user-event": "^13.5.0",
+    "@testing-library/dom": "^8.17.1",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^12.1.5",
```

Comment:
The V13 does not support react version < 17. We'll have to wait until we upgrade react.